app/models/email_authentication.rb

```diff
@@ -1,9 +1,24 @@
 class EmailAuthentication < ApplicationRecord
+  EXAMPLE_PASSPHRASE = "logic finite eager ratio"
+
   devise :database_authenticatable, :invitable, :lockable, :recoverable,
          :rememberable, :timeoutable, :trackable, :validatable, validate_on_invite: true
 
   has_one :user_authentication, as: :authenticatable
   has_one :user, through: :user_authentication
 
   delegate :full_name, :resources, :role, :organization, to: :user, allow_nil: true
+
+  validates :password, password_strength: {use_dictionary: true}, allow_nil: true
```

Comment:
Spaces inside the hash?

PS we talked about a styleguide tool that's friendlier than Rubocop. Might be a good idea to get that in sooner rather than later 😄 

Feel free to ignore my style-related comments until then if you'd like.